contracts/mocks/MasterMock.sol

```diff
@@ -10,8 +10,8 @@ contract MasterMock {
 
   enum Role {
     NonMember,
-    Member,
     AdvisoryBord,
```

Comment:
mispelling here

Reply:
Let's use `Unassigned` (capitalization) and `AdvisoryBoard` (missing "a").